ui/Dockerfile.dev

```diff
@@ -0,0 +1,24 @@
+FROM node:alpine3.16
+
+# Set working DIE
+WORKDIR /home/app/ui
+
+# Upgrade Alpine Packages
+RUN apk -U upgrade
+
+# Add bash to run tests and other scripts
+RUN apk add --no-cache bash
+
+# Copy package.json and package-lock.json
+COPY package*.json .
+
+# Install node packages
+RUN npm install
+
+# Copy rest of code
+COPY . .
+
+# PORT where UI will run in browser
+EXPOSE 3000
+
+CMD ["npm","run","dev","--","--host"]
```

Comment:
I would prefer `ENTRYPOINT` over `CMD` for the primary command that the container runs.